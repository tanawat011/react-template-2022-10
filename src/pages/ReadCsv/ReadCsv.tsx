export const ReadCsv = () => {
  // const onChange = () => {
  // const file = document.getElementById('file') as HTMLInputElement
  // const reader = new FileReader()
  // reader.onload = (e) => {
  //   const text = e.target?.result as string
  //   const lines = text.split('\n')
  //   lines.shift()
  //   lines.pop()
  //   const newLines = lines.map((line) => `bg-[${line.replaceAll('"', '').split(',')[0]}]`)
  //   console.log(newLines)
  // }
  // reader.readAsText(file.files![0])
  // }

  return (
    <>
      <p>ReadCsv</p>
      {/* <input id='file' type='file' onChange={onChange} /> */}
    </>
  )
}
