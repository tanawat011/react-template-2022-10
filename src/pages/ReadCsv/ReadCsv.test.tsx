import { render } from '@testing-library/react'

import { ReadCsv } from './ReadCsv'

describe('<ReadCsv />', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<ReadCsv />)
    expect(asFragment()).toMatchSnapshot()
  })
})
