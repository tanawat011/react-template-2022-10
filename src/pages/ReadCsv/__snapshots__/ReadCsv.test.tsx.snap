// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReadCsv /> match snapshot 1`] = `
<DocumentFragment>
  <p>
    ReadCsv
  </p>
  <input
    id="file"
    type="file"
  />
</DocumentFragment>
`;
